### PR TITLE
try punting emails to celery queue with no workers

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

**Review by dannyroberts — COMMENTED** [review]
Is the idea that we will keep this indefinitely, or that we not merge this and just deploy from this branch?

**Comment by nickpell** [comment]
I might suggest deploying this branch since it forks the current branch running on production, so we don't have to deal with any other changes since last deploy at this moment.

If it works, I'd say then merge this to master, so if anyone deploys master they won't bring down prod.

**Comment by dannyroberts** [comment]
What do we have to do not to mess up other envs, if we merge this to master?

**Comment by nickpell** [comment]
Updated to only affect staging (for testing) and prod

File: corehq/ex-submodules/dimagi/utils/django/email.py [hunk 1]
```diff
@@ -26,6 +27,29 @@
...
+            file_attachments=file_attachments, bcc=bcc)
+    else:
+        _send_HTML_email_run(subject, recipient, html_content, text_content=text_content,
+            cc=cc, email_from=email_from,
+            file_attachments=file_attachments, bcc=bcc)
+
+
+@task(queue="email_queue_punt",
+      bind=True, default_retry_delay=15 * 60, max_retries=10, acks_late=True)
+def _send_HTML_email_punt(subject, recipient, html_content, text_content=None,
```
Comment (emord) [inline corehq/ex-submodules/dimagi/utils/django/email.py]: When you have `bind=True` I think tha the first argument should be `self`

Comment (nickpell) [reply corehq/ex-submodules/dimagi/utils/django/email.py]: updated

**Pull request closed without merging**